src/get-started/modules/ROOT/pages/configuration-policies/directive.adoc

```diff
@@ -25,9 +25,9 @@ Let's use the filter in the directive tree to find it easily:
 
 image::./ssh.png["SSH technique", align="center"]
 
-Now click on "SSH server (OpenSSH)", the right part of the page will display the technique details, and
+Now click on *SSH server (OpenSSH)*, the right part of the page will display the technique details, and
 the list of available versions of the technique. We will use the latest one, by clicking on
-"Create with latest version".
+*Create with latest version*.
 
 This will make a new form appear, containing the configuration of the directive itself.
 As we said before, a technique is a parametrizable policy. A directive is a technique instance,
```

Comment:
I'm not sure about "parametrizable"

Reply:
I wasn't sure but it seems https://en.wiktionary.org/wiki/parametrizable

Reply:
I couldn't find the word in the dictionary, that's why :joy: